src/pkcs1v15.rs

```diff
@@ -458,15 +458,15 @@ mod tests {
         let signing_key = SigningKey::<Sha1>::new(priv_key);
 
         for (text, expected) in &tests {
-            let out = signing_key.sign(text.as_bytes()).to_bytes();
+            let sig: Signature = signing_key.sign(text.as_bytes());
```

Comment:
Is this really still needed?

Reply:
```
error[E0282]: type annotations needed
   --> src/pkcs1v15.rs:461:35
    |
461 |             let out = signing_key.sign(text.as_bytes()).to_bytes();
    |                                   ^^^^
    |
help: try using a fully qualified path to specify the expected types
    |
461 -             let out = signing_key.sign(text.as_bytes()).to_bytes();
461 +             let out = <pkcs1v15::signing_key::SigningKey<CoreWrapper<Sha1Core>> as Signer<S>>::sign(&signing_key, text.as_bytes()).to_bytes();
    |

For more information about this error, try `rustc --explain E0282`.
```

Without it.

In the end I think #505 creates less API breakage.

Reply:
I think the blanket impl is legitimately breaking inference for the case there's an explicit `Signer` impl instead of using the blanket one, which would be a case of https://github.com/RustCrypto/traits/issues/1831

Reply:
agreed